index.html

```diff
@@ -0,0 +1,244 @@
+                <div class="speaker-info">
+                    <h3>Zara Johnson</h3>
+                    <h6>CEO WaterBridges</h6>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                </div>
+
+            </div>
+            <div class="speaker3" id="">
+                <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles"> -->
+                <img src="./img/speaker-3.png" alt="Henry Steve" width="120vw">
+                <div class="speaker-info">
+                    <h3>Henry Steve</h3>
+                    <h6>Founder & CEO Cinder Inc.</h6>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+                </div>
+
+            </div>
+            <div class="speaker4" id="">
+                <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles"> -->
+                <img src="./img/speaker-4.png" alt="Lara James" width="120vw">
+                <div class="speaker-info">
+                    <h3>Lara James</h3>
+                    <h6>Founder & CEO Fire Myth</h6>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+                </div>
+
+            </div>
+            <div class="speaker3" id="">
+                <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles"> -->
+                <img src="./img/speaker-5.png" alt="Henry Steve" width="120vw">
+                <div class="speaker-info">
+                    <h3>Henry Steve</h3>
+                    <h6>Founder & CEO Cinder Inc.</h6>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
+                </div>
+
+            </div>
+            <div class="speaker4" id="">
+                <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles"> -->
+                <img src="./img/speaker-6.png" alt="Lara James" width="120vw">
+                <div class="speaker-info">
+                    <h3>Lara James</h3>
+                    <h6>Founder & CEO Fire Myth</h6>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                </div>
+
+            </div>
+        </div>
+    </section>
+
+
+    <section class="sponsors" id="sponsor">
+
+        <div class="sponsors-title">
+            <h5>Partner</h5>
+        </div>
+
+        <div class="logos">
+            <img src="./img/google.png" alt="Google" width="80vw">
+            <img src="./img/netflix.png" alt="Netflix" width="80vw">
+            <img src="./img/meta.png" alt="meta" width="80vw">
+            <img src="./img/uber.png" alt="Uber" width="80vw">
+            <img src="./img/cloudflare.png" alt="cloudflare" width="80vw">
+        </div>
+
+    </section>
+
+    <footer class="footer">
+        <div class="footer-block">
+            <h3>@2015 Creative Commons Korea <br>Inc. All Rights Reserved.</h3>
+        </div>
+    </footer>
+
```

Comment:
Kindly remove this section from the home page on mobile design as mentioned by the previous reviewer. In the original design, the sponsor section and the footer are not present on the home page in the mobile version they are on the about page. You can add an `id=hidden` to target it on the mobile screen e.g
 ```#hidden{display: none;```  
to hide this section on the mobile section.